abb_irb1200_gazebo/urdf/irb1200_5_90_macro.xacro

```diff
@@ -1,105 +1,93 @@
 <?xml version="1.0"?>
 <robot xmlns:xacro="http://ros.org/wiki/xacro">
-<xacro:include filename="$(find abb_irb1200_support)/urdf/irb1200_5_90_macro.xacro"/>
-<xacro:macro name="abb_irb1200_5_90_g" params="prefix">
+  <xacro:macro name="abb_irb1200_5_90_gazebo_spec" params="prefix">
 
-  <!-- get base ABB IRB1200 model -->
-  <xacro:abb_irb1200_5_90 prefix="${prefix}" />
+    <!-- transmission list -->
```

Comment:
Is this info just used for sim, or is it also potentially used by a motion planner taking into account control limits?  If so, seems like this would belong in the robot-specific URDF or the combined URDF?

Reply:
The `<transmission>` tag is only used by ros_control in gazebo for now. I followed the implementation of other abb_experimental gazebo packages, e.g. [irb120_3_58_macro.xacro](https://github.com/ros-industrial/abb_experimental/blob/e7e6c119c23ab257cb545450b927ed6df3920dcc/abb_irb120_gazebo/urdf/irb120_3_58_macro.xacro).